engines/groovie/script.cpp

```diff
@@ -746,13 +778,36 @@ bool Script::playvideofromref(uint32 fileref, bool loopUntilAudioDone) {
 	return true;
 }
 
+bool Script::playBackgroundSound(uint32 fileref, uint32 loops) {
+	if (fileref == -1) {
+		return false;
+	}
+
+	// Try to open the new file
+	Common::SeekableReadStream *_soundFile = _vm->_resMan->open(fileref);
+
+	if (_soundFile) {
+		_vm->_soundQueue.queue(_soundFile, loops);
+	} else {
+		error("Groovie::Script: Couldn't open file");
```

Comment:
`error()` is fatal in ScummVM, thus the `return` is unreachable. Or perhaps you wanted to use `warning()` here?